src/docker/ContextManager.ts

```diff
@@ -38,6 +38,7 @@ const defaultContext: Partial<DockerContext> = {
     Id: 'default',
     Name: 'default',
     Description: 'Current DOCKER_HOST based configuration',
+    Type: 'moby',

```

Comment:
The real bug was at line 202, the lack of setting `Type: 'moby'`. Setting it in the default context here makes sense since a `DOCKER_HOST`-based configuration is always `'moby'`, and it removes the need for the same line at line 250 below.